src/ui/components/ObjectView.vue

```diff
@@ -114,7 +118,7 @@ export default {
             this.currentView.show(this.viewContainer, this.openmct.editor.isEditing());
 
             if (immediatelySelect) {
-                this.removeSelectable = openmct.selection.selectable(
+                this.removeSelectable = this.openmct.selection.selectable(
```

Comment:
Good catch! This looks like a straight-up bug to me. Surprised it has not been throwing errors.